### DISCUSSION TITLE
V0.9.0

### DISCUSSION THREAD

**Discussion by mksunny1** [markersunny/eventiveness#1] [Announcements]
First version of Eventiveness released. Enjoy...

<hr /><em>This discussion was created from the release <a href='https://github.com/markersunny/eventiveness/releases/tag/'>V0.9.0</a>.</em>

**Comment by lolcabanon** [comment]
This looks super cool! I love the vibe of it already. I don't have the time to try it rn, but will surely do in the following weeks (or months... summer is coming!).

**Reply by mksunny1** [reply]
@lolcabanon glad to hear that. Please give your honest feedback when you do try it out and hopefully, you can get to make this even better. There is still a long way to go.
